src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -79,6 +81,31 @@ const DetailsModalStackNavigator = () => (
     </DetailsModalStack.Navigator>
 );
 
+const ParticipantsModalStackNavigator = () => (
+    <ParticipantsModalStack.Navigator
+        path={ROUTES.PARTICIPANTS}
```

Comment:
Pretty sure these path props are not doing anything and the "path" concept is left over from something we have since removed.